README.md

```diff
@@ -71,8 +71,8 @@ model Post {
   title     String
   content   String?
   published Boolean @default(false)
-  author    User?   @relation(fields:  [authorId], references: [id])
-  authorId  Int?
+  author    User    @relation(fields: [authorId], references: [id])
```

Comment:
The author being optional is actually intentional and matches the [Quickstart guide](https://www.prisma.io/docs/getting-started/quickstart-typescript#write-your-first-query-with-prisma-client).

Reply:
I will revert this and merge then :)